docs/database/cosmosdb.md

```diff
@@ -440,5 +448,743 @@ It captures the number of active instances at any given time. Best practices dic
 <!-- END AUTOGENERATED TEXT -->
 <!-- endsemconv -->
 
+## Network Level Metrics
+
+### Metric: `db.client.cosmosdb.request.duration`
+
+This metric is [recommended][MetricRecommended].
+
+It captures the end to end Latency of Azure Cosmos DB Backend network call.
+
+this metric SHOULD be specified with
+[`ExplicitBucketBoundaries`](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.35.0/specification/metrics/api.md#instrument-advisory-parameters)
+of `[ 0.001, 0.005, 0.01, 0.05, 0.1, 0.5, 1, 5, 10 ]`.
+
+<!-- semconv metric.db.client.cosmosdb.request.duration -->
+<!-- NOTE: THIS TEXT IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/snippet.md.j2 -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+
+| Name     | Instrument Type | Unit (UCUM) | Description    | Stability |
+| -------- | --------------- | ----------- | -------------- | --------- |
+| `db.client.cosmosdb.request.duration` | Histogram | `s` | End to end Latency of Azure Cosmos DB Backend network call | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
unit shoudl be ms not s

Reply:
Backend network call - IMO Backend is confusing here - maybe "service network call"

Reply:
check out https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/metrics.md#instrument-units:

> When instruments are measuring durations, seconds (i.e. `s`) SHOULD be used.